cmd/ratify/cmd/referrer.go

```diff
@@ -37,7 +36,7 @@ const (
 type referrerCmdOptions struct {
 	configFilePath string
 	subject        string
-	artifactTypes  []string
+	artifactType   string
```

Comment:
Hi @akashsinghal , i m not familiar with the referrer command, what is the previous sample usage for a  array of artifact type. Can you clarify how does it work with a single artifact type? thanks!